src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -342,7 +348,7 @@ public Boolean isOptional()
      * is probably a bad idea.
      * @param optional value to set
      */
-    public void setOptional(boolean optional) {
+    public void setOptional(Boolean optional) {
```

Comment:
`boolean` doesn't match the `get` method, so java's POJO logic doesn't pair them together; setting `Boolean` solves this, we box anyways, but now allow `null`

Reply:
nit: Might as well bump the brace to the next line while we're here?

Reply:
done